src/tests.rs

```diff
@@ -1,449 +1,424 @@
-#[cfg(test)]
```

Comment:
this module is already feature gated in `main.rs`, so we don't need it again here